orchagent/flexcounterorch.cpp

```diff
@@ -247,3 +251,76 @@ bool FlexCounterOrch::bake()
     Consumer* consumer = dynamic_cast<Consumer *>(getExecutor(CFG_FLEX_COUNTER_TABLE_NAME));
     return consumer->addToSync(entries);
 }
+
+QueuesState FlexCounterOrch::getQueueConfigurations(std::string portName)
+{
+    QueuesState queuesState{};
+    vector<FieldValueTuple> portFields;
+    m_counterQueueConfigTable.get(portName, portFields);
+    for (const auto& portField : portFields)
+    {
+        auto field = fvField(portField);
+        auto value = fvValue(portField);
+        if (field == "queue_list")
+        {
+            auto configQueuesState = tokenize(value, ',');
+            for (const auto& queue : configQueuesState)
+            {
+                int queueIndex{};
+                try {
+                    queueIndex = to_int<int>(queue);
+                } catch (std::invalid_argument const& e) {
+                    SWSS_LOG_WARN("Unable to parse string to int for [%s]", queue.c_str());
+                    continue;
+                }
+                if (isQueueIndexValid(queueIndex))
+                {
+                    queuesState.enableQueue(queueIndex);
+                }
+                else
+                {
+                    SWSS_LOG_WARN("Invalid counter queue index [%d] configured for port [%s]", queueIndex, portName.c_str());
+                }
```

Comment:
Makes more sense to use unsigned version:

```c++
template <typename T>
T to_uint(const std::string &str, T min = std::numeric_limits<T>::min(), T max = std::numeric_limits<T>::max())
```

Also, these functions already provide min/max bounds check, so I suggest:
```suggestion
                std::uint8_t queueIndex{};
                try {
                    queueIndex = to_int<std::uint8_t>(queue, MIN_QUEUE_INDEX, MAX_QUEUE_INDEX);
                } catch (std::invalid_argument const& e) {
                    SWSS_LOG_WARN("Invalid queue index [%s]", queue.c_str());
                    continue;
                }
```